### PR TITLE
Add Eicrud dataprovider to the docs

### PR DESCRIPTION
**Pull request by danyalutsevich**
Add Eicrud dataprovider to the docs

### CODE REVIEW

File: docs/DataProviderList.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ If you can't find a Data Provider for your backend below, no worries! [Writing a
 * ![mixer Logo](./img/backend-logos/github.svg "mixer Logo")**[Mixer](https://github.com/ckoliber/ra-data-mixer)**: [ckoliber/ra-data-mixer](https://github.com/ckoliber/ra-data-mixer)
 * ![moleculer Logo](./img/backend-logos/github.svg "moleculer Logo")**[Moleculer Microservices](https://github.com/RancaguaInnova/moleculer-data-provider)**: [RancaguaInnova/moleculer-data-provider](https://github.com/RancaguaInnova/moleculer-data-provider)
 * ![nestJs Logo](./img/backend-logos/nestjs.png "nestJs Logo")**[NestJS CRUD](https://github.com/nestjsx/crud)**: [rayman1104/ra-data-nestjsx-crud](https://github.com/rayman1104/ra-data-nestjsx-crud)
+* ![Eicrud Logo](./img/backend-logos/eicrud.svg "EiCrud Logo")**[Eicrud](https://github.com/eicrud/eicrud)**: [danyalutsevich/ra-data-eicrud](https://github.com/danyalutsevich/ra-data-eicrud)
```
Comment (fzaninotto) [inline docs/DataProviderList.md:58]: The data providers are sorted alphabetically in this list. Would you mind moving your insertion so it respects this order?

**Review by fzaninotto — CHANGES_REQUESTED** [review]

File: docs/DataProviderList.md (hunk 1 above)
Comment (danyalutsevich) [reply docs/DataProviderList.md]: Done

**Review by fzaninotto — APPROVED** [review]

**Comment by fzaninotto** [comment]
Thanks! Would you mind adding a License to your package?

**Comment by danyalutsevich** [comment]
sure

**Comment by danyalutsevich** [comment]
license added

**Pull request merged**